packages/react/src/stories/ic-date-input.stories.mdx

```diff
@@ -7,111 +7,134 @@ import {
-        />
-    </Story>
+  <Story name="Default">
+    <IcDateInput
+      label="When would you like to collect your coffee?"
+      onIcChange={(ev) => console.log(event.detail.value)}
+    />
+    <br />
+    <IcDateInput
+      label="When would you like to collect your coffee?"
+      dateFormat="MM/DD/YYYY"
+    />
+    <br />
+    <IcDateInput
+      label="When would you like to collect your coffee?"
+      dateFormat="YYYY/MM/DD"
+    />
+  </Story>
 </Canvas>
 
 export const DateInputWithValue = () => {
-    const dateInput = useRef(null);
-    const updateDate = () => {
-        dateInput.current.value = new Date('5 May 2005');
-    }
-    return (
-        <>
-          <IcDateInput
-            ref={dateInput}
-            id='date-input-default-date-with-value'
-            label='When would you like to collect your coffee?'
-            value='21/01/2001'
-          />
-          <IcButton onClick={updateDate}>Update date</IcButton>
-        </>
-    )
-}
-
-<Canvas>
-    <Story name="With value" parameters={{ loki: { skip: true } }}>
-      <DateInputWithValue />
-    </Story>
+  const dateInput = useRef(null);
+  const updateDate = () => {
+    dateInput.current.value = new Date("5 May 2005");
+  };
+  return (
+    <>
+      <IcDateInput
+        ref={dateInput}
+        id="date-input-default-date-with-value"
+        label="When would you like to collect your coffee?"
+        value="21/01/2001"
+      />
+      <IcButton onClick={updateDate}>Update date</IcButton>
+    </>
+  );
+};
+
+<Canvas>
+  <Story name="With value" parameters={{ loki: { skip: true } }}>
+    <DateInputWithValue />
+  </Story>
 </Canvas>
 
 ### Required
 
 <Canvas>
-    <Story name='Required'>
-      <IcDateInput
-        label='When would you like to collect your coffee?'
-        required
-      />
-    </Story>
+  <Story name="Required">
+    <IcDateInput label="When would you like to collect your coffee?" required />
+  </Story>
 </Canvas>
 
 ### Disabled
 
 <Canvas>
-    <Story name='Disabled'>
-      <IcDateInput
-        label='When would you like to collect your coffee?'
-        disabled
-      />
+  <Story name="Disabled">
+    <IcDateInput label="When would you like to collect your coffee?" disabled />
+  </Story>
+</Canvas>
+
+### With calendar
+
+<Canvas>
+    <Story name='With calendar'>
+        <IcDateInput
+          label='When would you like to collect your coffee?'
+          showCalendarButton
+        />
     </Story>
 </Canvas>
 
-### Custom helper text
+### Sizes
 
 <Canvas>
-    <Story name='Custom helper text'>
-      <IcDateInput
-        label='When would you like to collect your coffee?'
-        helperText='We will have your order ready for you on this day'
-      />
+    <Story name='Sizes'>
+        <IcDateInput
+          label='When would you like to collect your coffee?'
+          size="small"
+        />
+        <br/>
+        <IcDateInput
+          label='When would you like to collect your coffee?'
```

Comment:
Should there be a size="medium" or for it to be called size="default" I enquire this due to consistency with other components such as [Loading indicator](https://design.sis.gov.uk/components/loading-indicators).

Reply:
I think this is right as it aligns with the input component container

Reply:
Maybe thoughts should be drawn out for size consistency between components.